bio/rbt/fastq_split/wrapper.py

```
@@ -0,0 +1,71 @@
+        f"This wrapper required at least {required_threads + len(files_to_compress)} "
+        f"threads. Got {snakemake.threads}."
+    )
+
+if len(files_to_compress) > 0:
+    compression_threads = get_compression_threads(
+        available_threads=snakemake.threads - required_threads,
+        compression_jobs=len(files_to_compress),
+    )
+    command_lines.append(f"mkfifo {files_to_compress.keys()} ;")
+
+    crabz_params = zip(files_to_compress.items(), compression_threads)
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)